### PR TITLE
suggest atoum/stubs ( #508 )

### LINKED ISSUE #508: Suggest atoum/stubs for dependency?
I think it could be interesting to suggest in composer to add atoum/stubs.

What's your advice on this ?

### PR DESCRIPTION
**Pull request by Grummfy**
fix #508

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
 		"classmap": [ "classes/" ]
 	},
 	"suggest": {
-		"ext-mbstring": "Provides support for UTF-8 strings"
+		"ext-mbstring": "Provides support for UTF-8 strings",
+		"atoum/stubs": "Provides stubs for atoum that will help ide for autocompletion"
```
Comment (Hywan) [inline composer.json]:
```diff
- ide
+ IDE
```
Actually, the whole sentence is not very clearn.
What about:
> Provides IDE support (like autocompletion)

This is simple but it's understandable.

Comment (jubianchi) [inline composer.json]: +1 with @Hywan

**Comment by Grummfy** [comment]
How can I improve this?

File: composer.json (hunk 1 above)
Comment (mikaelrandy) [inline composer.json]: :+1: too

**Comment by jubianchi** [comment]
@Grummfy just change the wording and it should be good ;) @Hywan's suggestion seems good to me.

**Comment by jubianchi** [comment]
oh yeah! :+1:

**Comment by Hywan** [comment]
:+1:

**Pull request merged**